shopify-imagemagick@6.rb

```diff
@@ -5,8 +5,8 @@ class ShopifyImagemagickAT6 < Formula
   # ImageMagick site removes tarballs regularly which means we get issues
   # unnecessarily and older versions of the formula are broken.
   url "https://dl.bintray.com/homebrew/mirror/imagemagick%406-6.9.9-40.tar.xz"
-  mirror "https://www.imagemagick.org/download/ImageMagick-6.9.9-40.tar.xz"
-  sha256 "62f25d46bfbcffe00b84e167994d593868a9a47d4defc489a429cae1d6aab9f7"
+  mirror "https://download.imagemagick.org/ImageMagick/download/releases/ImageMagick-6.9.9-51.tar.xz"
```

Comment:
The line with "URL" should be changed, too.

Reply:
Given that we lost the `-40` version, we should host this tarball somewhere that we control, so it can't be lost again. The imagemagick address should be used for url, and then our host for mirror.

Reply:
Both `url` and `mirror` where invalid. 

Not quite sure what else to put here.. `bintray` is deprecated.

Reply:
This https://github.com/Shopify/homebrew-shopify/releases/tag/bag-of-holding release exists to hold binaries for this repo.  Edit it and add the binary. GitHub will give you a url to use for the mirror.